packages/ozone/src/api/admin/emitModerationEvent.ts

```diff
@@ -2,156 +2,204 @@ import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
 import { Server } from '../../lexicon'
 import AppContext from '../../context'
 import {
+  isModEventDivert,
   isModEventEmail,
   isModEventLabel,
   isModEventReverseTakedown,
   isModEventTakedown,
 } from '../../lexicon/types/com/atproto/admin/defs'
+import { HandlerInput } from '../../lexicon/types/com/atproto/admin/emitModerationEvent'
 import { subjectFromInput } from '../../mod-service/subject'
 import { ModerationLangService } from '../../mod-service/lang'
 import { retryHttp } from '../../util'
+import { ModeratorOutput, RoleOutput } from '../../auth-verifier'
+
+const handleModerationEvent = async ({
```

Comment:
refactored this into a function so that we can execute the entire flow again after divert event.